backend/src/services/sumo_access/parameter_access.py

```diff
@@ -20,17 +22,30 @@
 class ParameterAccess(SumoEnsemble):
     async def get_parameters_and_sensitivities(self) -> EnsembleParameters:
         """Retrieve parameters for an ensemble"""
+        timer = PerfTimer()
 
-        # Sumo query. Replace with explorer when ready
-        sumo_parameters: List[SumoEnsembleParameter] = await get_parameters_for_iteration(
-            self._sumo_client, self._case_uuid, self._iteration_name
+        table_collection = self._case.tables.filter(
+            iteration=self._iteration_name,
+            aggregation="collection",
+            name="parameters",
+            tagname="all",
         )
+        if await table_collection.length_async() == 0:
+            raise ValueError(f"No parameter tables found {self._case.name, self._iteration_name}")
+        if await table_collection.length_async() > 1:
+            raise ValueError(f"Multiple parameter tables found {self._case.name,self._iteration_name}")
 
-        sensitivities = create_ensemble_sensitivities(sumo_parameters)
-        parameters = [create_ensemble_parameter(sumo_parameter) for sumo_parameter in sumo_parameters]
+        table = table_collection[0]
+        byte_stream: BytesIO = table.blob
```

Comment:
Should this be async in some way?

Reply:
There is a table.blob_async which do return a BytesIO object, but the following lines:
```python
        table = await table_collection.getitem_async(0)
        byte_stream: BytesIO = await table.blob_async
        table = pq.read_table(byte_stream)
```
gives: `pyarrow.lib.ArrowInvalid: Could not open Parquet input source '<Buffer>': Parquet magic bytes not found in footer. Either the file is corrupted or this is not a parquet file.`

Not entirely sure why...